torch/distributed/fsdp/_runtime_utils.py

```diff
@@ -481,24 +482,25 @@ def _root_pre_forward(
     )
     _clear_grads_if_needed(_all_handles(state))
 
+    # Prepares the forward inputs by moving them to ``device``
```

Comment:
Small nit: since for `state`, the device is saved as `compute_device` (for historical reasons)
```suggestion
    # Prepares the forward inputs by moving them to `compute_device`
```

(I am guessing the \`\`device\`\` comes from the `_prepare_forward_inputs` docstring.)